MvvmHelpers/Exceptions/InvalidCommandParameterException.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace MvvmHelpers.Exceptions
+{
+	/// <summary>
+	/// Represents errors that occur during IAsyncCommand execution.
+	/// </summary>
+	public class InvalidCommandParameterException : Exception
+	{
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:MvvmHelpersInvalidCommandParameterException"/> class.
+		/// </summary>
+		/// <param name="expectedType">Expected parameter type for AsyncCommand.Execute.</param>
+		/// <param name="actualType">Actual parameter type for AsyncCommand.Execute.</param>
+		/// <param name="innerException">Inner Exception</param>
+		public InvalidCommandParameterException(Type expectedType, Type actualType, Exception innerException) : base(CreateErrorMessage(expectedType, actualType), innerException)
+		{
+
+		}
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:MvvmHelpers.InvalidCommandParameterException"/> class.
+		/// </summary>
+		/// <param name="expectedType">Expected parameter type for AsyncCommand.Execute.</param>
+		/// <param name="actualType">Actual parameter type for AsyncCommand.Execute.</param>
+		public InvalidCommandParameterException(Type expectedType, Type actualType) : base(CreateErrorMessage(expectedType, actualType))
+		{
+
+		}
+
+		static string CreateErrorMessage(Type expectedType, Type actualType) =>
```

Comment:
Add two more constructors here to align with this feedback: https://github.com/jamesmontemagno/mvvm-helpers/pull/43#discussion_r315461010

```csharp
/// <summary>
/// Initializes a new instance of the <see cref="T:TaskExtensions.MVVM.InvalidCommandParameterException"/> class.
/// </summary>
/// <param name="excpectedType">Excpected parameter type for AsyncCommand.Execute.</param>
/// <param name="innerException">Inner Exception</param>
public InvalidCommandParameterException(Type excpectedType, Exception innerException) : base(CreateErrorMessage(excpectedType), innerException)
{

}

/// <summary>
/// Initializes a new instance of the <see cref="T:TaskExtensions.MVVM.InvalidCommandParameterException"/> class.
/// </summary>
/// <param name="excpectedType">Excpected parameter type for AsyncCommand.Execute.</param>
public InvalidCommandParameterException(Type excpectedType) : base(CreateErrorMessage(excpectedType))
{

}

static string CreateErrorMessage(Type excpectedType) => $"Invalid type for parameter. Expected Type {excpectedType}";
```